content/en/logs/indexes/_index.md

```diff
@@ -78,6 +78,33 @@ Update or remove this quota at any time when editing the Index:
 
 **Note**: Indexes daily quotas reset automatically at 2:00pm UTC (4:00pm CET, 10:00am EDT, 7:00am PDT).
 
+### Attribute based sampling
+
+You can sample on a specific attribute holding an ID, such as User, Session, Request Identifiers, in addition to the default log-based sampling (applied when the exclusion percentage is set on `all logs`).
+
+This feature operates on attribute's values rather than the log itself: for example, given an attribute `id` holding 3 distinct values (`1`, `2`, `3`), defining a 66% exclusion filter on it, results in indexing all logs holding 1 specific attribute's value (`3`).
+
+{{< img src="logs/indexes/index_exclusion_attribute_groups.png" alt="index attribute-based exclusion filter groups"  style="width:70%;">}}
+
+For example, exclude `nginx` logs for 99% of `http.request_id`:
+
+{{< img src="logs/indexes/index_exclusion_on_attribute.png" alt="index attribute-based exclusion filter"  style="width:70%;">}}
+
+**Note**: 
+
+* You must prefix the attribute name with the `@` character.
+
+* For low cardinality attributes, you should define a set of distinct Index Exclusion Filters on `all logs`.
```

Comment:
Let's use passive format. Something like:

* Attribute name must be prefixed by `@` (i.e if the attribute is called `http.request_id`, in the exclusion filter it must be `@http.request_id`)
* The attribute-based sampling is recommended for high cardinality attributes (like session_id, trace_id, ...)